ms/applink.c

```diff
@@ -34,9 +34,12 @@
 #define APPLINK_MAX     22      /* always same as last macro */
 
 #ifndef APPMACROS_ONLY
+
+#ifndef APPLINK_NOiNCLUDES
```

Comment:
The mixed case looks very strange to me. I would just call this `APPLINK_NO_INCLUDES`